apps/src/maze/maze.js

```diff
@@ -175,6 +178,11 @@ module.exports = class Maze {
         dom.addClickTouchEvent(finishButton, this.finishButtonClick_);
       }
 
+      if (this.controller.skin.adjustableSpeed) {
+        var slider = document.getElementById('slider');
+        this.speedSlider = new Slider(10, 35, 130, slider);
```

Comment:
Can these be pulled out as constants? Would we ever want to change them? And for my own curiosity- what do they mean?

Reply:
This injects a slider into an SVG in the maze visualization if the skin has adjustableSpeed true. We store the slider in this.speedSlider so that at execution time, we can get the slider's value. 

And yes, slider can be made a constant, fixing that now :)

Reply:
Oh nice. And what are the 10, 35, and 130 for?

Reply:
Within the SVG, it is at x and y location (10, 35) with a width of 130 pixels